src/makeRobotAccountant.js

```diff
@@ -1,12 +1,28 @@
 'use strict';
 
-/**
+/*
+ *
  *
  * @return {function}
  */
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      counter++;
+
+      if (counter > 3) {
+        return counter % 2 === 0
+          ? 'Bzzz... Error!'
+          : firstNum + secondNum;
+      }
+
+      return firstNum + secondNum;
```

Comment:
```suggestion
     return counter > 3 && counter % 2 === 0
        ? 'Bzzz... Error!'
        : firstNum + secondNum;
```
You can do it shorter and more readable:)

Reply:
reasonable, thx